package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adr-log",
```

Comment:
@sergy-gzh I'd suggest changing `name` key in `package.json` to `@deity/adr-log` (since we may refer to this package leaving the original package name untouched)